pixi.toml

```diff
@@ -94,6 +94,9 @@ ros-humble-foxglove-bridge="*"
 # ros-humble-rqt = "*"
 # ros-humble-test-msgs = "*"
 
+ros-humble-dynamixel-sdk = "*"
```

Comment:
any particular reason this isn't version-floored?

Reply:
I tried to floor it to the current version but that caused an error, I'll try to use a lower version.

Reply:
resolved in slack.